metrology/reporter/statsd.py

```diff
@@ -101,11 +128,9 @@ def _send_tcp(self):
     def _send_udp(self):
         if len(self.batch_buffer):
             if sys.version_info[0] > 2:
-                self.socket.sendto(bytes(self.batch_buffer, 'ascii'),
-                                   (self.host, self.port))
+                self.socket.sendto(self.batch_buffer, (self.host, self.port))
```

Comment:
there is no difference between py2 and py3 now - shouldn't you use `bytes` here?